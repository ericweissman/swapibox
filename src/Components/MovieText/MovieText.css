aside {
  background-color: black;
  font-family: 'Star Jedi';
  color: yellow;
}
/*# sourceMappingURL=MovieText.css.map */