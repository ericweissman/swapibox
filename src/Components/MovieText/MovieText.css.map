{
    "version": 3,
    "mappings": "AAAA,AAAA,KAAK,CAAC;EACJ,gBAAgB,EAAE,KAAK;EACvB,WAAW,EAAE,WAAW;EACxB,KAAK,EAAE,MAAM;CACd",
    "sources": [
        "MovieText.scss"
    ],
    "names": [],
    "file": "MovieText.css"
}