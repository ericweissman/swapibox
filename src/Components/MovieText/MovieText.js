import React from 'react';
import '../../Main.scss';
import PropTypes from 'prop-types'


const MovieText = (props) => {
  const {title, year, crawl} = props.films
  return(
    <aside>
      <h2>{title}</h2>
      <h3>{year}</h3>
      <p>{crawl}</p>
    </aside>
  )
}

MovieText.propTypes = {
  films: PropTypes.shape({
    title: PropTypes.string,
    year: PropTypes.string,
    crawl: PropTypes.string
  })
}

export default MovieText;