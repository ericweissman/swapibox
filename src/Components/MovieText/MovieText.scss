aside {
  font-family: $title-font;
  background-color: $menus;
  color: $accent;
  text-align: center;
  min-height: 700px;
  display: flex;
  flex-direction: column;
  align-items: center;
  h2 {
    font-size: 35px;
    animation: slideInY 8s ease 0s 1 normal; 
    animation-fill-mode: forwards;
  }
  h3 {
    margin: 0;
    animation: slideInY 8s ease 0s 1 normal; 
    animation-fill-mode: forwards;
  }
  p {
    font-family: $text-font;
    width: 285px;
    font-size: 20px;
    line-height: 30px;
    color: $text;
    animation: slideInY 8s ease 0s 1 normal; 
    animation-fill-mode: forwards;
  }
}

@keyframes slideInY {
  0% {
    transform: translateY(500px);
  }
  100% {
    transform: translateY(0px);
  }
}