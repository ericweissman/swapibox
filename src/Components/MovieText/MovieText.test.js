import React from 'react';
import MovieText from './MovieText'
import { shallow } from 'enzyme'

describe('MovieText', () => {
  const films = {title: 'Title', crawl: 'Text', year: '1996'}

  it('should match snapshot', () => {
   const wrapper = shallow(<MovieText films={films} />)
   expect(wrapper).toMatchSnapshot();
  });
}) 