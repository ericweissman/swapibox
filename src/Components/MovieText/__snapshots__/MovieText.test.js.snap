// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieText should match snapshot 1`] = `
<aside>
  <h2>
    Title
  </h2>
  <h3>
    1996
  </h3>
  <p>
    Text
  </p>
</aside>
`;
